pallets/remote-asset-manager/Cargo.toml

```diff
@@ -65,10 +65,10 @@ cumulus-pallet-xcm = { git = "https://github.com/paritytech/cumulus", branch = "
 parachain-info = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.7" }
 
 # polkadot
-xcm-builder = { git = 'https://github.com/paritytech/polkadot', branch = 'release-v0.9.7', features = ['runtime-benchmarks'] }
+xcm-builder = { git = 'https://github.com/paritytech/polkadot', branch = 'release-v0.9.7' }
 polkadot-runtime-parachains = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.7" }
 polkadot-parachain = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.7" }
-pallet-xcm = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.7", features = ['runtime-benchmarks'] }
+pallet-xcm = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.7" }
```

Comment:
I've added the features here so that we can. compile it with `--all-features`

Reply:
Once we use these pallets/dependencies with `runtime-benchmarks` feature, we should add them to `dependencies` with `optional = true`

Reply:
I see, that makes sense.